modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -2,24 +2,62 @@
 
-. Download Anypoint Studio from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site] and extract it into the 'C:\' root folder. +
+[NOTE]
+Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
+
+After downloading and installing the JDK, ensure that your JDK installation is properly configured within the the `JAVA_HOME` environment variable by following the instructions below.
+
+. In Search, search for `System (Control Panel)` and select it.
+. Click *Advanced System Settings*.
+. Click *Environment Variables*.
+. In *System Variables*, select the `PATH` environment variable.
+. Click *Edit*.
++
+If the `PATH` environment variable does not exist, click *New*.
+. In *Edit System Variable* (or *New System Variable*), ensure that the JDK 1.8.0 `/bin` directory is the first item in your `PATH` environment variable, and click *OK*. +
+Blank spaces are not allowed when defining a path on Windows.
+. Click *OK*.
+
+== Installation
+
+. Extract the Windows installation file into the `C:\` root folder. +
 If you are using a Windows Anti-Virus application, ensure that the `plugin/` and `features/` directories are under the “trusted” category.
 . Open the extracted Anypoint Studio.
 . Click OK to accept the default workspace. +
-+
-[NOTE]
---
-You cannot reuse a workspace used by Anypoint Studio 6.x or below. +
-If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
---
+You cannot reuse a workspace used by Anypoint Studio 6.x or below. If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
+
+== Troubleshooting
 
-Anypoint Studio displays a welcome message when it loads for the first time. +
-Click Get Started to start using Anypoint Studio.
+If Studio fails to load with a *An Error Has Occured* message, you must configure Anypoint Studio to point to your JDK installation as the default VM.
+
+. Locate your `AnypointStudio.ini` file inside your Anypoint Studio's installation directory.
+. Add the -vm option to point to your specific JDK installation path.
++
+[source,text,linenums]
+----
+-vm
+C:\progra~1\Java\jdk1.8.0_73\bin\javaw.exe
+----
++
+* Blank spaces are not allowed when defining paths on Windows. This is why `\Program Files\` changes to `\progra~1\`.
+* The `-vm` option and the installation path must be in separate lines.
+* The value must be the full absolute or relative path to the Java executable, not just to the Java home directory.
+* The `-vm` option must occur after the other Studio-specific options (such as `-product`, `--launcher.*`, etc), but before the `-vmargs` option, because everything after `-vmargs` is passed directly to the JVM.
++
+If you encounter “Java was started but returned exit code=1” error while starting Anypoint Studio, modify the `-vm` argument to point to the `jvm.dll` as following:
```

Comment:
If you encounter “Java was started but returned exit code=1” while starting Anypoint Studio, modify the `-vm` argument to point to the `jvm.dll':

(removes "error" and "as following"_